resources/dashboard:Alarm-Trend-Dashboard.json

```diff
@@ -0,0 +1,97 @@
+{
+   "migrationVersion": {
+      "dashboard": "7.0.0"
+   }, 
+   "references": [
+      {
+         "type": "visualization", 
+         "id": "Count-of-Total-Alarms-Fired-by-Name-(Bar-Chart)", 
+         "name": "panel_0"
+      }, 
+      {
+         "type": "search", 
+         "id": "Alarms-Table", 
+         "name": "panel_1"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "Total-Alarms-Fired-(Count)", 
+         "name": "panel_2"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "103ccf70-ff2e-11e9-b97c-935442fb8ac3", 
+         "name": "panel_3"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "bb4b8d20-ff2e-11e9-b97c-935442fb8ac3", 
+         "name": "panel_4"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "d5d7ef50-ffdb-11e9-b97c-935442fb8ac3", 
+         "name": "panel_5"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "e6262110-ffdb-11e9-b97c-935442fb8ac3", 
+         "name": "panel_6"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "f3f9b5e0-ffdb-11e9-b97c-935442fb8ac3", 
+         "name": "panel_7"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "b85f5490-ff2c-11e9-b97c-935442fb8ac3", 
+         "name": "panel_8"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "d3541e20-ff2c-11e9-b97c-935442fb8ac3", 
+         "name": "panel_9"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "1f0d5750-ff2d-11e9-b97c-935442fb8ac3", 
+         "name": "panel_10"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "14f67c60-ff2d-11e9-b97c-935442fb8ac3", 
+         "name": "panel_11"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "Summary-of-Total-Alarms-Fired-(Pie-Chart)", 
+         "name": "panel_12"
+      }, 
+      {
+         "type": "visualization", 
+         "id": "8a62e1e0-0019-11ea-a805-f766c8673742", 
+         "name": "panel_13"
+      }
+   ], 
+   "type": "dashboard", 
+   "dashboard": {
+      "hits": 0, 
+      "timeFrom": "now-7d", 
```

Comment:
@craig-cogdill I left it because it's the Alarm trend dashboard and they had it saved. If it's only showing the last 15 minutes it's not really showing a trend. Keeping that in mind, do you still want to removed the saved timeframe?